bundles/org.openhab.binding.omnilink/README.md

```diff
@@ -0,0 +1,321 @@
+| heat_setpoint               | Number:Temperature   | The current low/heating setpoint of this thermostat.                                 | thermostat                          |
+| cool_setpoint               | Number:Temperature   | The current high/cooling setpoint of this thermostat.                                | thermostat                          |
+| humidify_setpoint           | Number:Dimensionless | The current low/humidify setpoint for this thermostat.                               | thermostat                          |
+| dehumidify_setpoint         | Number:Dimensionless | The current high/dehumidify setpoint for this thermostat.                            | thermostat                          |
+| system_mode                 | Number               | The current system mode of this thermostat.                                          | thermostat                          |
+| fan_mode                    | Number               | The current fan mode of this thermostat.                                             | thermostat                          |
+| hold_status                 | Number               | The current hold status of this thermostat.                                          | thermostat                          |
+| status                      | Number               | The current numeric status of this thermostat/unit/dimmable unit.                    | thermostat, unit, dimmable          |
+| level                       | Dimmer               | Increase/Decrease the level of this unit/dimmable unit/UPB unit.                     | unit, dimmable, upb                 |
+| switch                      | Switch               | Turn this unit/dimmable unit/flag/output/room on/off.                                | unit, dimmable, flag, output, room  |
+| on_for_seconds              | Number               | Turn on this unit for a specified number of seconds.                                 | unit, dimmable, upb, flag, output   |
+| off_for_seconds             | Number               | Turn off this unit for a specified number of seconds.                                | unit, dimmable, upb, flag, output   |
+| on_for_minutes              | Number               | Turn on this unit for a specified number of minutes.                                 | unit, dimmable, upb, flag, output   |
+| off_for_minutes             | Number               | Turn off this unit for a specified number of minutes.                                | unit, dimmable, upb, flag, output   |
+| on_for_hours                | Number               | Turn on this unit for a specified number of hours.                                   | unit, dimmable, upb, flag, output   |
+| off_for_hours               | Number               | Turn off this unit for a specified number of hours.                                  | unit, dimmable, upb, flag, output   |
```

Comment:
Maybe?  But it would be tricky. They are mapping directly to command of the same type on the backend system. 

So if you get a Number:Time of 90 seconds, you need to know to send it to seconds and not send 1 to minutes and then 30 seconds. 

And not time values are valid. 
So if you get a time of like 15 hours and 8 minutes and 9 seconds.  You can't just convert that to seconds because it would exceed the number of seconds you can send. So you would have to ignore anything beyond the hours at that point which might confuse the user.

Reply:
Sounds tricky. But sounds tricky for the user, too. I won't insist on changing it.

Reply:
I'm inclined to leave it as it is a lot of extra work. If I'm feeling adventurous at a later date I may try to implement this